test/e2e/data/shared/v1beta1_provider/metadata.yaml

```diff
@@ -18,3 +18,7 @@ releaseSeries:
   - major: 1
     minor: 7
     contract: v1beta1
+  - major: 1
```

Comment:
I think this should also be in a main folder (cf the same in core CAPI)

Reply:
I went over and back on this - providers seem to follow a different pattern, but I'm fine to change it.

Reply:
I think we will need something like this once we add the upgrade test as this one should require a different metadata.yaml

Reply:
Same for the "shared/v1beta1/metadata.yaml" (just noticed)